satpy/tests/test_composites.py

```diff
@@ -90,6 +91,7 @@ def test_mult_ds_diff_area(self):
     def test_mult_ds_diff_dims(self):
         """Test that datasets with different dimensions still pass."""
         from satpy.composites import CompositeBase
+
```

Comment:
Again; why is there an empty line added in lines 94 and 107, but not between lines 79 and 80 on 179 and 180?